store/app/routers/stripe.py

```diff
@@ -340,13 +338,13 @@ class CreateConnectAccountResponse(BaseModel):
     account_id: str
 
 
-@stripe_router.post("/create-connect-account", response_model=CreateConnectAccountResponse)
+@stripe_router.post("/connect/account", response_model=CreateConnectAccountResponse)
 async def create_connect_account(
     user: Annotated[User, Depends(get_session_user_with_read_permission)],
     crud: Annotated[Crud, Depends(Crud.get)],
 ) -> CreateConnectAccountResponse:
     try:
-        logger.info(f"[CREATE-CONNECT] Starting new account creation for user {user.id}")
+        logger.info(f"Starting new account creation for user {user.id}")
```

Comment:
nit: prefer not to use f-strings in logging
```suggestion
        logger.info("Starting new account creation for user %s", user.id)
```